scripts/cleanup_clusters/cleanup_clusters.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+
+ACTION=$1
+
+if [[ "$ACTION" != "list" && "$ACTION" != "delete" ]]
+then
+    echo "***ERROR***"
+    echo "Please choose 'list' or 'delete' actions and try again."
+    echo "***ERROR***"
+
+    exit 1
+fi
+
+
+# TODO: quality.firecloud.org ?
```

Comment:
We'll want to run this on the QA domain eventually.

Reply:
maybe make this a list of test + quality and check membership?

Reply:
Yes - will need to test with a quality user